packages/client/src/forms/configuration/administrative/utils.ts

```diff
@@ -504,7 +505,7 @@ export function getPlaceOfEventLocationSelect(
         operation: 'eventLocationQueryTransformer',
         parameters: [
           locationIndex,
-          location,
+          location as keyof Address,
```

Comment:
`keyof Address` doesn't include `'locationLevel3' | 'locationLevel4' | 'locationLevel5'`. This is an existing bug and we can maybe fix it later in a separate PR. `eventLocationQueryTransformer` also takes in `keyof Address`, but [here](https://github.com/opencrvs/opencrvs-core/blob/fe8fcbc7d50a10027e5998a66c456c7a59deb135/packages/client/src/forms/configuration/administrative/addresses.ts#L661) we pass those locationLevels to this function 🤔 @Zangetsu101